examples/basic/main.tf

```diff
@@ -14,13 +8,11 @@ module "example" {
   name        = "tracing"
   attributes  = ["foo"]
 
-  aws_account_id                  = local.aws_account_id
-  aws_region                      = local.aws_region
   queue_name                      = "myQueue"
   alarm_create                    = true
-  alarm_topic_arn                 = "arn:aws:sns:eu-central-1:${local.aws_account_id}:alarm"
+  alarm_topic_arn                 = "arn:aws:sns:eu-central-1:123456789123:alarm"
   alarm_minutes                   = 5
   alarm_evaluation_periods        = 3
   alarm_period                    = 60
-  principals_with_send_permission = ["arn:aws:iam::${local.foreign_aws_account_id}:root"]
+  principals_with_send_permission = ["arn:aws:iam::123456789123:root"]
```

Comment:
Now you do repeat yourself + you added a bug.

Reply:
the `alarm_topic_arn` and the `principals_with_send_permission` were in different accounts before. Now you made them be in the same account.